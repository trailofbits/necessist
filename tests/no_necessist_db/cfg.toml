args = ["--root", "examples/cfg"]

[bin]
name = "necessist"

[status]
code = 1

[env.add]
TRYCMD="1"
