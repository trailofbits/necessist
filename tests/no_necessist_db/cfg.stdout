4 candidates in 1 test file
examples/cfg/src/lib.rs: dry running
examples/cfg/src/lib.rs: mutilating
